doc/content/source/kernels/ConcreteThermalConduction.md

```diff
@@ -1,14 +1,21 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # ConcreteThermalConduction
 
-!alert construction title=Undocumented Class
-The ConcreteThermalConduction has not been documented. The content contained on this page includes the
-typical automatic documentation associated with a MooseObject; however, what is contained is
-ultimately determined by what is necessary to make the documentation clear for users.
-
 !syntax description /Kernels/ConcreteThermalConduction
 
+## Description
+
+This kernel provides thermal conductivity part in the governing partial differential equation for heat transfer in concrete is given by [!cite](bazant1982finite) and [!cite](saouma_structural_2014) as
```

Comment:
Missing punctuation. Also, maybe use 'heat conduction' instead of 'thermal conductivity.